### DISCUSSION TITLE
【新人爆照】

### DISCUSSION THREAD

**Discussion by shxtxdy** [share121/inter-knot#2725] [其他]
哎呀，不小心把我变身时的照片发出来了，不处😋
![36ef6c0a03ca5a45894ea7cb039df7be1d193822](https://github.com/user-attachments/assets/72932431-04b8-4d46-9a0f-60d15ddab9c8)

**Comment by hanhanhan12** [comment]
😡

**Comment by Maple013** [comment]
滴嘟——
前新艾利都治安局治安官提醒您，保护个人隐私，谨防信息泄露，请安全使用绳网，不要公开自己的照片、身份信息等个人隐私！

**Comment by WFGoose** [comment]
怎么放尸体啊，捂眼.jpg<img src="https://img-static.mihoyo.com/communityweb/upload/9939adf214795d97dad9abd90265aab2.png" alt="帕姆_哭哭" width="75" >

**Comment by MrDucker** [comment]
光之孙笑川<img src="https://bbs-static.miyoushe.com/static/2024/07/31/213614cd6275ec54d80fe9f84a0edf62_6334766189957723007.png" alt="白露 灭火" width="75" >

**Comment by XiaopuWARER** [comment]
禁止抛尸！！！

**Comment by zhixuan45** [comment]
怎么有人抛尸啊ewe

**Comment by Yulooooo** [comment]
谁屙这儿了

**Comment by wu364** [comment]
天天抛尸<img src="https://img-static.mihoyo.com/communityweb/upload/9886c96ebc61d1513fbeb0b10c95ba35.gif" alt="吃我一炮" width="75" >

**Comment by Zhitong2107** [comment]
绳网虽然是法外之地，但也不许抛尸，臭味隔着屏幕都闻到了。。。
![IMG_20240818_202612](https://github.com/user-attachments/assets/68c4d44b-f0ae-4382-9caf-88abae7b7c74)
